CorePayments/src/main/java/com/paypal/android/corepayments/api/FetchClientToken.kt

```diff
@@ -0,0 +1,44 @@
+package com.paypal.android.corepayments.api
+
+import androidx.annotation.RestrictTo
+import com.paypal.android.corepayments.APIClientError
+import com.paypal.android.corepayments.APIRequest
+import com.paypal.android.corepayments.CoreConfig
+import com.paypal.android.corepayments.HttpMethod
+import com.paypal.android.corepayments.RestClient
+import com.paypal.android.corepayments.base64encoded
+import org.json.JSONObject
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class FetchClientToken(
+    private val coreConfig: CoreConfig,
+    private val restClient: RestClient = RestClient(configuration = coreConfig),
+) {
+    suspend operator fun invoke(): String {
```

Comment:
```suggestion
    suspend fun createToken(): String {
```
We mostly use the functor pattern for UseCase type classes. I am interested in this pattern. If we decide to go this route, I'd like for all of the other `API` suffixed classes to migrate to this pattern so we can keep it consistent.

Reply:
renaming to `getToken()` because same token is returned after it's created

Reply:
It isn't a new token every time? Also, I went with "create" in the naming prefix because the underlying HTTP method is a `POST`.